app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -42,14 +42,16 @@ def perform(form526_submission_id, full_name)
 
     private
 
-    def send_fast_track_stakeholder_email_for_testing(form526_submission_id)
+    def send_fast_track_engineer_email_for_testing(form526_submission_id)
       # TODO: This should be removed once we have basic metrics
       # on this feature and the visibility is imporved.
+      body = "A claim was just submitted on the #{Rails.env} environment " \
+             "with submission id: #{form526_submission_id} and job_id #{jid}"
```

Comment:
I'm not seeing where jid is coming from.

Reply:
good catch! I just fixed it. It should have been `self.jid`

Reply:
Hmm, it looks like self. isn't actually needed. My bad.